WebUI/generate.sh

```diff
@@ -88,4 +83,11 @@ make html
 if [ $? -ne 0 ]; then
   echo -e "Failed to generate documentation.\n"
   exit 2
+fi
+
+mv $BUILD_DIR/_build/html $BUILD_DIR/../../${PROJECTNAME}_preview && cd $_/../
```

Comment:
where is this being moved to? $BUILD_DIR can be at depth > 1. In that case i think, $BUILD_DIR/../../ would point to a different directory.